### PR TITLE
Feature/mttl 901 bootstrap

### PR DESCRIPTION
**Pull request by mikemoody-amido**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/MTTL-901

## Describe this PR

Bootstrap shell listener application for activity history

#### _Checklist_

- [x] Code pipeline builds correctly

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+
+obj/
+out/
+bin/
+.idea/
+tmp/
+.vs/
+/notes-api.Tests/Properties/launchSettings.json
```
Comment (humulla) [inline .gitignore:62]: This should presumably change to `tenure-listener`

File: ActivityListener/.lambda-test-tool/SavedRequests/Person Update.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "Records": [
+    {
+      "messageId": "19dd0b57-b21e-4ac1-bd88-01bbb068cb78",
+      "receiptHandle": "MessageReceiptHandle",
+      "body": "{\"id\": \"8e648f3d-9556-4896-8400-211cb1c5451b\",\"eventType\": \"PersonCreatedEvent\",\"sourceDomain\": \"Person\",\"sourceSystem\": \"PersonAPI\",\"version\": \"v1\",\"correlationId\": \"f4d541d0-7c07-4524-8296-2d0d50cb58f4\",\"dateTime\": \"2021-05-17T11:59:57.25Z\",\"user\":{\"id\": \"ac703d87-c100-40ec-90a0-dabf183e7377\",\"name\": \"Joe Bloggs\",\"email\": \"joe.bloggs@hackney.gov.uk\"},\"entityId\": \"30ea9335-841f-4b59-ab76-bdba3a305cda\"}",
```
Comment (humulla) [inline ActivityListener/.lambda-test-tool/SavedRequests/Person Update.json:6]: I'm guessing the eventType and sourceDomain values would change

File: ActivityListener/.lambda-test-tool/SavedRequests/PersonUpdate.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "Records": [
```
Comment (humulla) [inline ActivityListener/.lambda-test-tool/SavedRequests/PersonUpdate.json:2]: Why do we have 2 of the same files?

Comment (mikemoody-amido) [reply ActivityListener/.lambda-test-tool/SavedRequests/PersonUpdate.json:2]: These saved requests shouldn't be checked in - I will remove.

File: ActivityListener/.lambda-test-tool/SavedRequests/Person Update.json (hunk 2 above)
Comment (mikemoody-amido) [reply ActivityListener/.lambda-test-tool/SavedRequests/Person Update.json:6]: These saved requests shouldn't be checked in - I will remove.

**Review by humulla — APPROVED** [review]

**Pull request merged**